web/setup.py

```diff
@@ -2,13 +2,13 @@
 
 setup(
     name='qresp',
-    version='1.2.0',
+    version='1.2.1',
     url='http://qresp.org/',
     entry_points = {
         'console_scripts': ['qresp=project.__main__:main'],
     },
     license='GNU',
-    author='Aditya Tanikanti, Marco Govoni',
+    author='Aditya Tanikanti, Marco Govoni, Sushant Bansal',
```

Comment:
Change the order of the authors. SB, AT, MG

Reply:
Done